dll/win32/msgina/resources/license/FancyLogOffDlg.txt

```diff
@@ -0,0 +1,23 @@
+The following bitmaps/icons are made by Splitwirez (Splitwirez@gmail.com
+
+).
```

Comment:
This seems a bit weird?

Reply:
@learn-more This is weird, I didn't have this issue earlier.

Reply:
The line head has space. Delete them.